DataAnnotationsValidator/DataAnnotationsValidator/DataAnnotationsValidator.nuspec

```diff
@@ -1,16 +0,0 @@
-﻿<?xml version="1.0"?>
-<package >
-  <metadata>
-    <id>DataAnnotationsValidator</id>
-    <version>$version$</version>
-    <authors>Mike Reust</authors>
-    <owners>Mike Reust</owners>
-    <licenseUrl>https://github.com/reustmd/DataAnnotationsValidatorRecursive/blob/master/License.md</licenseUrl>
-    <projectUrl>https://github.com/reustmd/DataAnnotationsValidatorRecursive</projectUrl>
-    <requireLicenseAcceptance>false</requireLicenseAcceptance>
-    <description>Allows scanning object graph and validates every object using .NET standatd System.ComponentModel.DataAnnotations.Validator</description>
-    <copyright>Copyright © 2016</copyright>
-    <tags>DataAnnotation validation validator</tags>
-    <dependencies />
-  </metadata>
-</package>
```

Comment:
why are you killing the `nuspec`?

Reply:
I moved all the information contained in this file to `DataAnnotationsValidator.csproj` file